tensorflow/core/protobuf/autotuning.proto

```diff
@@ -70,6 +70,46 @@ message AutotuneResult {
   // Next ID: 14
 }
 
+message AutotuneExecutionPlanResult {
```

Comment:
Can you merge this AutotuneResult? AutotuneResult already supports Convs and Gemms, it should be no problem supporting a third type of key (e.g. `CudaConvPlan` containing an id).

Reply:
Done.